src/Commands/stubs/Caddyfile

```diff
@@ -0,0 +1,38 @@
+{
+	{$CADDY_GLOBAL_OPTIONS}
+
+	frankenphp {
+		worker vendor/laravel/octane/bin/frankenphp-worker.php {$WORKER_COUNT}
+	}
+}
+
+{$SERVER_NAME} {
+	log {
+		# Redact the authorization query parameter that can be set by Mercure
+		format filter {
+			wrap console
+			fields {
+				uri query {
+					replace authorization REDACTED
+				}
+			}
+		}
+	}
+
+	route {
+		encode zstd gzip
+
+		# Mercure config is injected here
+		{$CADDY_SERVER_EXTRA_DIRECTIVES}
+
+		# If the requested file does not exist, pass to the worker script
+		@indexFiles file {
+			try_files public/{path} vendor/laravel/octane/bin/frankenphp-worker.php
+			split_path .php
+		}
+		rewrite @indexFiles {http.matchers.file.relative}
+		@phpFiles path *.php
+		php @phpFiles
```

Comment:
This can be replaced with `php_server`, no? Can do this:

```
php_server {
	try_files {path} {path}/ /frankenphp-worker.php
}
```

Assumes that the worker script is moved to the `public/` webroot.

Reply:
Currently, we can't (see https://github.com/laravel/octane/pull/764#discussion_r1375212084), but if we change the design to publish the worker script in `public/` then indeed we'll be able to simplify the config.

Reply:
done